All project code/index.js

```diff
@@ -196,6 +196,207 @@ app.get("/profile", (req, res) => {
     }});
 });
 
+
+app.get('/visit', (req, res) => {
+    res.render("pages/visit");
+});
+
+// The fields to render on the search page to make the search request.
+const weatherFields = [
+    // name is the js parameter name to be used in POST requests. Label is what is shown to the user.
+    {name: "startTime", label: "Start Time", required: true},
+    {name: "endTime", label: "End Time", required: true},
+    {name: "locationLattitude", label: "Location Lattitude", required: true},
+    {name: "locationLongitude", label: "Location Longitude", required: true},
+    {name: "requestParameters", label: "Request Parameters", required: true},
+    {name: "dataFormat", label: "Data Format", required: true},
+    {name: "optionalParameters", label: "Optional Parameters", required: false},
+];
+
+// Display search page
+app.get("/search", (req, res) => {
+    res.render("pages/search", {
+        searchFields: weatherFields
+    });
+});
+
+/**
+ * Generates a complete request URL to send to meteomatics with various parameters
+ * @param {string} startTime Start time for the time-range of data request.
+ * - Format (ISO 8601), in UTC: YYYY-MM-DDThh:mm:ssZ
+ * - Up to 24h in the past. 
+ * @param {string} endTime End time for the time-range of data request. 
+ * - Format(ISO 8601), in UTC: YYYY-MM-DDThh:mm:ssZ
```

Comment:
I get what you're trying to do here, but this isn't super readable for people who haven't seen this format before (it's also not in the scope of the tech stack we're using and you typically don't define normal functions this way, this sort of syntax is usually reserved for defined endpoints that a user can actually call or it gets confusing)

Reply:
It's the JSDoc standard, and I felt it was important to include detailed comments in this form because it would get very messy without it. And these are the comments that explain exactly how the API needs to be used, with the types and exact format very important. 
So in my case this documentation is meant for the users of the API, which is me and others working on the search feature. This will all get abstracted away for the end users so they don't need to worry about it, later.

Reply:
You can get extensions that auto-format these comments when you hover over the function, or where it may be getting called. It's just a little more convenient and clean this way.